csharp/Platform.Setters/SetterBase.cs

```diff
@@ -6,10 +6,13 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>Represents a base implementation for an setter that allows you to set a passed value as the result value.</para>
-    /// <para>Представляет базовую реализацию для установщика, который позволяет установить переданное ему значение в качестве результирующего значения.</para>
+    /// <para>Provides a base implementation for an setter that allows you to set a passed value as the result value.</para>
```

Comment:
`Provides a base class that allows you to set a passed value as the result value.`